Dynatrace/dynatrace-operator
# Automate creation of EEC service

## Description

This PR creates a service reconclier and service to communicate between OTelC and EEC.


## How can this be tested?

1. Deploy dynakube `v1beta3` with prometheus enabled, and check that service has been created or deleted if you remove dynakube.
